src/app/shuffle/shuffle-styled.js

```diff
@@ -55,4 +55,27 @@ export const EntryThumbnail = styled.div`
   aspect-ratio: 16 / 9;
   background: ${({ $backgroundImage }) => `no-repeat center / cover url(${$backgroundImage})`};
   width: 150px;
+`;
+
+export const ControlButton = styled.button`
```

Comment:
We'll want to style our already existing `Button` component here with

`export const ControlButton = styled(Button)`